apps/newsletters-ui/src/app/components/views/DraftListView.tsx

```diff
@@ -1,31 +1,35 @@
 import AddIcon from '@mui/icons-material/Add';
-import { Box, Container } from '@mui/material';
-import { useLoaderData } from 'react-router-dom';
+import { Box, Button, Container } from '@mui/material';
+import { useLoaderData, useNavigate } from 'react-router-dom';
 import { isDraft } from '@newsletters-nx/newsletters-data-client';
 import { DraftsTable } from '../DraftsTable';
-import { NavigateFab } from '../NavigateFab';
 
 export const DraftListView = () => {
 	const list = useLoaderData();
+	const navigate = useNavigate();
 	if (!list || !Array.isArray(list)) {
-		return <nav>NO LIST</nav>;
+		return <nav>No Drafts</nav>;
 	}
 
 	const drafts = list.filter(isDraft);
 	return (
 		<>
 			<DraftsTable drafts={drafts} />
 			<Container maxWidth="lg">
-				<Box paddingX={1} display={'flex'} justifyContent={'flex-end'}>
-					<NavigateFab
-						href="/newsletters/newsletter-data"
-						color="primary"
-						variant="extended"
+				<Box
+					paddingX={1}
+					paddingBottom={'8px'}
```

Comment:
```suggestion
					paddingBottom={1}
```
using `1` here will give us the default MUI padding of 8px - I think the padding size is configurable.